webpack.config.prod.js

```diff
@@ -39,6 +39,7 @@ module.exports = Object.assign(prodConfig, {
     filename: "[name]-[chunkhash].js",
     chunkFilename: "[id]-[chunkhash].js",
     crossOriginLoading: "anonymous",
+    publicPath: "",
```

Comment:
See https://stackoverflow.com/questions/65245185/new-to-webpackautoprefix-problem-with-webpack-manifest-plugin/65272040#65272040

Reply:
Setting `publicPath` to an empty string works fine when assets are being served from the same directory structure. What about consider aligning it with how it's done in mitxonline, as seen [here](https://github.com/mitodl/mitxonline/blob/main/frontend/public/webpack.config.js#L32). In our case, it could be set to `/static/bundles/` for xPRO.

We should also ensure consistency in the dev configuration. For dev, `publicPath `could be set to `process.env.PUBLIC_PATH`, and PUBLIC_PATH can be defined in the Docker [environment](https://github.com/mitodl/mitxpro/blob/master/docker-compose.yml#L79-L82), e.g., `PUBLIC_PATH: http://${MIT_XPRO_HOSTNAME:-xpro.odl.local}:8053/`.

Reply:
@Anas12091101, we have a similar PR here: https://github.com/mitodl/mitxpro/pull/3112 that relates to the change. I think the remaining issue can be addressed there.